README.md

```diff
@@ -119,6 +119,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Tim Delisle](people/tim-delisle.md)
 * [Vivek Sudarsan](people/vivek-sudarsan.md)
 * [Yanbo Li](people/yanbo-li.md)
+* [Anas Bouzoubaa](people/anas-bouzoubaa.md)
```

Comment:
We've sorted these alphabetically. Would you mind doing that please and putting your name towards the top? Make sure to _squash_ multiple commits, too.